specs/latest/2.0/index.html

```diff
@@ -725,40 +732,91 @@ <h2 class="no-toc">Table of contents</h2>
   void texStorage2D(GLenum target, GLsizei levels, GLenum internalformat, GLsizei width, GLsizei height);
   void texStorage3D(GLenum target, GLsizei levels, GLenum internalformat, GLsizei width, GLsizei height,
                     GLsizei depth);
-  void texImage2D(GLenum target, GLint level, GLint internalformat, GLsizei width, GLsizei height,
-                  GLint border, GLenum format, GLenum type, GLintptr offset);
-  void texSubImage2D(GLenum target, GLint level, GLint xoffset, GLint yoffset, GLsizei width, GLsizei height,
-                     GLenum format, GLenum type, GLintptr offset);
+
+
+  void texImage2D(GLenum target, GLint level, GLint internalformat,
+                  GLsizei width, GLsizei height, GLint border, GLenum format,
+                  GLenum type, GLintptr offset);
+  void texImage2D(GLenum target, GLint level, GLint internalformat,
+                  GLsizei width, GLsizei height, GLint border, GLenum format,
+                  GLenum type, ByteBuffer? pixels,
+                  optional unsigned long byteOffset = 0);
```

Comment:
No reason to give these size args, since we don't enforce anything but that the source buffer has enough bytes.

glCompressedTexImage* traditionally does have a size arg so that it can safety-check that the size matches exactly, but I was never sure why that was added to the spec.